yarpctest/context_test.go

```diff
@@ -67,7 +67,8 @@ func TestContextWithCall(t *testing.T) {
 				RoutingDelegate: "routingdelegate",
 				ResponseHeaders: tt.resHeaders,
```

Comment:
Thanks for the change.
I am not sure if it changes anything (maybe better verbose)... since map[string]string will is a reference... call.WriteResponseHeader("baz", "qux") updated result will be visible on tt.resHeaders as well
can you add a test-case where this change affects are shown?